examples/pipeline_wavernn/main.py

```diff
@@ -0,0 +1,411 @@
+        type=int,
+        metavar="N",
+        help="print frequency in epochs",
+    )
+    parser.add_argument(
+        "--batch-size", default=256, type=int, metavar="N", help="mini-batch size"
+    )
+    parser.add_argument(
+        "--learning-rate", default=1e-4, type=float, metavar="LR", help="learning rate",
+    )
+    parser.add_argument("--clip-grad", metavar="NORM", type=float, default=4.0)
+    parser.add_argument(
+        "--mulaw",
+        default=True,
+        action="store_true",
+        help="if used, waveform is mulaw encoded",
+    )
+    parser.add_argument(
+        "--jit", default=False, action="store_true", help="if used, model is jitted"
+    )
+    parser.add_argument(
+        "--upsample-scales",
+        default=[5, 5, 11],
+        type=List[int],
+        help="the list of upsample scales",
+    )
+    parser.add_argument(
+        "--n-bits", default=8, type=int, help="the bits of output waveform",
+    )
+    parser.add_argument(
+        "--sample-rate",
+        default=22050,
+        type=int,
+        help="the rate of audio dimensions (samples per second)",
+    )
+    parser.add_argument(
+        "--hop-length",
+        default=275,
+        type=int,
+        help="the number of samples between the starts of consecutive frames",
+    )
+    parser.add_argument(
+        "--win-length",
+        default=1100,
+        type=int,
+        help="the number of samples between the starts of consecutive frames",
+    )
+    parser.add_argument(
+        "--f-min",
+        default=40.0,
+        type=float,
+        help="the number of samples between the starts of consecutive frames",
+    )
+    parser.add_argument(
+        "--min-level-db",
+        default=-100,
+        type=float,
+        help="the min db value for spectrogam normalization",
+    )
+    parser.add_argument(
+        "--n-res-block", default=10, type=int, help="the number of ResBlock in stack",
+    )
+    parser.add_argument(
+        "--n-rnn", default=512, type=int, help="the dimension of RNN layer",
+    )
+    parser.add_argument(
+        "--n-fc", default=512, type=int, help="the dimension of fully connected layer",
+    )
+    parser.add_argument(
+        "--kernel-size",
+        default=5,
+        type=int,
+        help="the number of kernel size in the first Conv1d layer",
+    )
+    parser.add_argument(
+        "--n-freq", default=80, type=int, help="the number of bins in a spectrogram",
+    )
+    parser.add_argument(
+        "--n-hidden", default=128, type=int, help="the number of hidden dimensions",
+    )
+    parser.add_argument(
+        "--n-output", default=128, type=int, help="the number of output dimensions",
+    )
+    parser.add_argument(
+        "--n-fft", default=2048, type=int, help="the number of Fourier bins",
+    )
+    parser.add_argument(
+        "--mode",
+        default="waveform",
+        choices=["waveform", "mol"],
+        type=str,
+        help="the mode of waveform",
```

Comment:
Having waveform in the help may be confusing since `waveform` is one of the option. Please rephrase.

Reply:
This essentially decides what loss function to use, right? Either `CrossEntropyLoss` or `discretized mixture of logistic distributions loss`. This would be clearer as a setting.

Reply:
How about calling this `--loss`, and then having choices: `CrossEntropy` and `MoL` instead?

Reply:
I like the `--loss` name. But `mode` is also a parameter of wavernn model. In wavernn model, each mode has a different output channel (`n_classes = 2 ** n_bits` or `n_classes = 30`).

Reply:
Let's change to `--loss` and then please open a separate pull request to address [comment](https://github.com/pytorch/audio/pull/735/files#r456477181) beforehand.

Reply:
It has been updated and a separate PR is #797 .